checks/permissions_test.go

```diff
@@ -94,8 +94,8 @@ func TestGithubTokenPermissions(t *testing.T) {
 				Error:         nil,
 				Score:         checker.MaxResultScore,
 				NumberOfWarn:  0,
-				NumberOfInfo:  3,
-				NumberOfDebug: 3,
+				NumberOfInfo:  2,
+				NumberOfDebug: 4,
```

Comment:
The reason there is one less info and more debug log (here and in `package workflow write`) is because `requiresContentsPermissions` now returns false.